invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/modules/modal.overrides

```diff
@@ -126,3 +126,10 @@
     border-color: @mutedTextColor transparent transparent;
   }
 }
+
+
+.access-link-modal.scrolling.content {
+  overflow-y: auto !important;
+  overflow-x: hidden !important;
```

Comment:
Instead of hiding the x-overflow, does it work to e.g. set the `max-width` of the table to `100%`? To make sure we don't crop content.